vioscsi/helper.c

```diff
@@ -56,21 +56,25 @@ SendSRB(
     UCHAR               ScsiStatus = SCSISTAT_GOOD;
     ULONG MessageID;
     int res = 0;
+    BOOLEAN             notify = FALSE;
     PREQUEST_LIST       element;
     ULONG               index;
-ENTER_FN_SRB();
 
-    if (!Srb)
-        return;
+ENTER_FN_SRB();
 
+    if (!Srb) {
```

Comment:
Seemed like a sanity check to me. It lives on, so maybe someone saw one in the past. We can add a WPP macro here to check and use a reserved flag and just look for that in the trace. It would take pretty extreme performance testing, and more to the point - fairly thorough fail condition testing. Maybe raise a new issue?

Reply:
Right, it is a kind of sanity check. Static driver verifier was failing without this check.